### DISCUSSION TITLE
Sample code for Java Web App integration

### DISCUSSION THREAD

**Discussion by wcl-onespan** [sevdokimov/log-viewer#59] [Q&A]
Hi,
Is there a small sample java web app with this project integrated that someone can share?  I have written a few java web apps, but for some reason I am not able to figure out where to begin with this library.

Thanks
-Will

**Comment by sevdokimov** [comment]
@wcl-onespan , do you use spring or some other framework?

**Reply by wcl-onespan** [reply]
Yes Spring would be fine.  I tried to take the demo code you had in the /demos/ directory, but I could not figure out how to use that code to embed your code into my spring web app.  Specifically I couldn't figure out how to modify the config to limit view of specific directories and files, and I wanted to embed the app in a general dashboard, so the viewer would be one component of the dashboard with a html header above it.  Obviously I dont need the sample code to do all that, but the sample code that is up there was not enough for me to figure out how to get started with embedding it in a spring project.
